docs/BACKBEAT_ROUTES.md

```diff
@@ -0,0 +1,152 @@
+# Backbeat routes
+
+Backbeat routes are implemented in `lib/routes/routeBackbeat.js`.
+
+This special router is responsible for handling all the requests that are
+related to the Backbeat service. Backbeat may call any of the below APIs to
+perform operations on either data or s3 objects (metadata).
+
+These route follow the same authorization and validation as the S3 routes:
+
+- Authorize the request with support for Implicit Denies from the IAM service.
+- Retrieve the bucket and object metadata if applicable.
+- Evaluate the S3 Bucket Policies and ACLs before authozing the request.
+  - Backbeat routes are only authorized given the right permission, currently,
+    `objectReplicate` as a unique permission for all these special routes.
+  - In order to be authorized without S3 Bucket Policy, the caller must be
+    authorized by the IAM service and the ACLs. Service accounts and accounts
+    are allowed.
+- Finally, evaluate the quotas before allowing the request to proceed.
+
+## List of supported APIs
+
+```plaintext
+PUT /_/backbeat/metadata/<bucket name>/<object key>
+```
+
+To edit one existing S3 Object's metadata.
+
+```plaintext
+GET /_/backbeat/metadata/<bucket name>/<object key>?versionId=<version id>
+```
+
+To get one existing S3 Object's metadata. Version id can be specified to get
+the metadata of a specific version.
+
+```plaintext
+PUT /_/backbeat/data/<bucket name>/<object key>
+```
+
+To put directly to the storage layer the data for an existing S3 Object.
+
+```plaintext
+PUT /_/backbeat/multiplebackenddata/<bucket name>/<object key>?operation=putobject
+```
+
+To put directly to the storage layer the data for an existing S3 Object.
+Use case: Cross Region Replication (CRR).
```

Comment:
All `multiplebackenddata` APIs are used by Zenko Replication and not CRR.
CRR uses the two following APIs:
- PUT /_/backbeat/metadata/<bucket name>/<object key>
- GET /_/backbeat/metadata/<bucket name>/<object key>?versionId=<version id>
- PUT /_/backbeat/data/<bucket name>/<object key>
as well as 
- POST /_/backbeat/batchdelete
which is used by other features as well.